src/amo/components/HeroRecommendation/index.js

```diff
@@ -73,11 +74,16 @@ export class HeroRecommendationBase extends React.Component<InternalProps> {
   };
 
   onHeroClick = () => {
-    const { _tracking } = this.props;
+    const { _tracking, shelfData } = this.props;
+
+    invariant(shelfData, 'The shelfData property is required');
+
+    const { addon } = shelfData;
 
     _tracking.sendEvent({
-      action: this.makeCallToActionURL(),
+      action: PRIMARY_HERO_CLICK_ACTION,
       category: PRIMARY_HERO_CLICK_CATEGORY,
+      label: addon ? addon.guid : undefined,
```

Comment:
Should we send anything if there is no `addon`?

Reply:
I think we'd still want to know the hero was clicked. This just means the hero was for an external item and not an add-on. I don't see why we would want to *not* send it.

Reply:
ah ok, I see. I think you're right. Can we have `external` or a more specific value tha none?

Reply:
Good idea. I've updated this.